src/common/transformations/include/transformations/common_optimizations/mark_exp_reduceop_to_keep_in_mixed_precision.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API MarkExpReduceOpToKeepInMixedPrecision;
+
+}  // namespace pass
+}  // namespace ov
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief: MarkExpReduceOpToKeepInMixedPrecision  marks path that goes
```

Comment:
```suggestion
 * @brief: MarkExpReduceOpToKeepInMixedPrecision marks path that goes
```

Reply:
removed extra space from the comment in a new place of this pass